eclipse-glsp/glsp-theia-integration
# GLSP-1022: Update dependencies

- Update to latest glsp-client next version
- Update to inversify 6 and fix API breaks
- Update to current Theia next version to consistently use inversify 6. Once the 1.39 release is available we should switch to that
- Use Typescript 5 Part of https://github.com/eclipse-glsp/glsp/pull/1022
